### ISSUE TITLE
music resets upon playing sound

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#399]
reproduce:
enter the game and bump to wall during music.. music will reset and start from the beginning

1) tried with only mp3 files inside of folders
2) added both .mp3 and .ogg files https://github.com/igroglaz/Tangaria_release/commit/5589f043134e3e2e9bb48f7ee9c594f6bab58cfa

**Comment by s88100** [comment]
`Mix_PlayChannel(-1, sample->sample_data.chunk, 0));`  plays only 'ogg' sounds (SDL_CHUNK)

**Comment by igroglaz** [comment]
[14:55] Serega88: @igroglaz  checked Windows 10 (1903) latest Tangaria_release, sounds and music play fine. There maybe a problem with sound card or Windows version.
in your case, mp3 sounds are selected to the music channel  Mix_PlayMusic(sample->sample_data.music, 1));  (case SDL_MUSIC)

thanks! I've checked - it was my old client which didn't had libogg-0.dll libvorbis-0.dll libvorbisfile-3.dll files.. after I've added them - all works
